src/components/_cards/EthBtcPerfomanceCard.tsx

```diff
@@ -33,14 +39,70 @@ export const EthBtcPerfomanceCard: VFC<BoxProps> = (props) => {
   const id = useRouter().query.id as string
   const cellarConfig = cellarDataMap[id].config
   const tokenPrice = useTokenPrice(cellarConfig)
+  const isLarger768 = useBetterMediaQuery("(min-width: 768px)")
   const [timeline, setTimeline] = useState<string>("1W")
+  const [pointActive, setPointActive] = useState<Point>()
+
+  const MobileTooltip = () => {
+    if (!!pointActive && !isLarger768) {
+      const { id: pointId, serieId } = pointActive
+      const [_, i] = pointId.split(".")
+      return (
+        <Stack
+          p={4}
+          bg="rgba(18, 18, 20, 0.8)"
```

Comment:
I'll leave the call up to you, but it may be worth adding this color to our chakra theme and calling that string here instead. Especially if this semi-transparent bg is used elsewhere. Not critical to PR merge, though!

Reply:
good idea, will call that color `surface.blackTransparent` for future uses